circle.yml

```diff
@@ -1,3 +1,3 @@
 machine:
   post:
-    - pyenv global 2.7.12 3.6.2
+    - pyenv global 3.6.12 3.7.9 3.8.5
```

Comment:
Is this file needed? We don't use Circle for this lib anymore.

Reply:
Left it from the previous version in case anyone wants to use it.

Reply:
In case it would be merged back to upstream...

Reply:
I think that's not going to happen, seems it has been abandoned.